modules/ROOT/nav.adoc

```diff
@@ -241,8 +241,8 @@
 * xref:securing.adoc[Securing]
  ** xref:anypoint-enterprise-security.adoc[Anypoint Enterprise Security]
   *** xref:installing-anypoint-enterprise-security.adoc[Installing Anypoint Enterprise Security]
-  *** link:/mule-user-guide/v/3.6/mule-secure-token-service[Mule Secure Token Service]
-   **** link:/mule-user-guide/v/3.6/creating-an-oauth-2.0-web-service-provider[Creating an Oauth 2.0 Web Service Provider]
+  *** xref:3.8@mule-secure-token-service.adoc[Mule Secure Token Service]
```

Comment:
This goes to Mule 3.8 version from a 3.5 navigation.
Do we really want to do that? I think we should remove this line.